includes/dashboard.php

```diff
@@ -713,14 +713,13 @@ function action_admin_enqueue_dashboard_scripts() {
 
 		$data = array( 'nonce' => wp_create_nonce( 'ep_dashboard_nonce' ) );
 
+		$wpcli_sync = (bool) Utils\is_indexing_wpcli();
```

Comment:
No longer need to cast this since the function signature is bool and it returns a casted bool already.

```suggestion
		$wpcli_sync = Utils\is_indexing_wpcli();
```